types/express-serve-static-core/index.d.ts

```diff
@@ -29,6 +29,8 @@ import { EventEmitter } from 'events';
 import { Options as RangeParserOptions, Result as RangeParserResult, Ranges as RangeParserRanges } from 'range-parser';
 import { ParsedQs } from 'qs';
 
+export {};
```

Comment:
Is this needed? 🤔

Reply:
yeah, otherwise the linter complains because `GetRouteParameter` isn't exported